### PR TITLE
sign up page with connection to aws cognito

### LINKED ISSUE #65: connect front end sign up page with backend

### PR DESCRIPTION
**Pull request by IAfanasovMob**
closes #65 and #63

### CODE REVIEW

**Comment by IAfanasovMob** [comment]
closes #63

**Comment by IAfanasovMob** [comment]
> Looks great. Is there a plan for translating all of the texts in English as well?

sure. after MVP

File: src/views/Entrepreneur.vue [hunk 1]
```diff
@@ -122,6 +115,7 @@ import axios from 'axios'
 import FormField from '@components/FormField.vue'
 import EntrepreneurFlowIntro from '@components/entrepreneur-flow/EntrepreneurFlowIntro.vue'
 import EntrepreneurFlowTermsAndConditions from '@components/entrepreneur-flow/EntrepreneurFlowTermsAndConditions.vue'
+// import AmplifyEventBus from 'aws-amplify'
```
Comment (cezaryskura) [inline src/views/Entrepreneur.vue]: do we need this line?

Comment (IAfanasovMob) [reply src/views/Entrepreneur.vue]: nope. will delete as part of some another commit because any change in the branch will clear pr approvals

**Pull request merged**